.github/workflows/docs.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and Deploy
+on: [push, pull_request]
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.1 # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
+        with:
+          persist-credentials: false
+
+      - name: Dependencies
+        run: |
+          .github/workflows/dependencies/documentation.sh
+          python3 -m pip install sphinx sphinx_rtd_theme breathe
+
+      - name: Install and Build
+        run: |
+          ./build_docs.sh
+
+      - name: Deploy
+        if: github.event_name == 'push' && github.repository == 'AMReX-Codes/amrex' && github.ref == 'refs/heads/development'
+        uses: JamesIves/github-pages-deploy-action@3.7.1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ACCESS_TOKEN: ${{ secrets.DEPLOY_DOCS }}
+          REPOSITORY_NAME: atmyers/atmyers.github.io
```

Comment:
Is this the right repo?

Reply:
Er, no. I was testing it on mine and forgot to update.